app/src/main/java/com/yeonae/chamelezone/data/repository/place/PlaceRepositoryImpl.kt

```diff
@@ -68,34 +68,49 @@ class PlaceRepositoryImpl private constructor(private val remoteDataSource: Plac
         remoteDataSource.getKeyword(callBack)
     }
 
-    override fun modifyPlace(
+    override fun updatePlace(
+        placeNumber: Int,
+        images: List<String>,
         memberNumber: Int,
-        keywordNames: List<Int>,
-        name: String,
         address: String,
-        openingTimes: List<String>,
         phoneNumber: String,
         content: String,
         latitude: BigDecimal,
         longitude: BigDecimal,
-        images: List<String>,
+        imageNumber: List<Int>,
```

Comment:
```suggestion
        imageNumbers: List<Int>,
                imageNumberList: List<Int>,
```
복수개를 의미하는 네이밍으로 변경이 필요합니다.

Reply:
수정했습니다
915f192
307dc97